pkg/server/http/http.go

```diff
@@ -75,7 +75,14 @@ func (s *Server) ListenAndServe() error {
 	if err != nil {
 		return errors.Wrap(err, "server could not be started")
 	}
-	return errors.Wrap(toolkit_web.ListenAndServe(s.srv, s.opts.tlsConfigPath, s.logger), "serve HTTP and metrics")
+
+	flags := &toolkit_web.FlagConfig{
```

Comment:
I wonder if we should just import flags from `exporter-toolkit` and pass those options. There is https://pkg.go.dev/github.com/prometheus/exporter-toolkit@v0.8.1/web/kingpinflag#AddFlags

Reply:
I think that would introduce more changes. I can do that in a separate followup PR. WDYT?

Reply:
Yup, follow-up sounds good!